templates/page.tpl.php

```diff
@@ -87,18 +87,15 @@
 </main>
 
 <footer role="contentinfo">
-  <section class="page-footer">
-    <div class="wrapper">
-      <section class="footer-top">
-        <nav>
-          <?php print render($page['footer']); ?>
-        </nav>
-      </section>
-    </div>
-  </section>
-  <section class="page-bottom">
-    <div class="wrapper">
+  <div class="wrapper">
+    <section class="footer-top">
+      <?php print render($page['footer_top']); ?>
+    </section>
+    <section class="footer-bottom">
+      <?php print render($page['footer_bottom']); ?>
```

Comment:
With the footer bottom, do we use the layout in http://guides.service.gov.au/design-guide/patterns/navigation/index.html#footer-navigation . 
`<section>
      <div class="footer-logo">
        <img alt="Australian Government Coat of Arms" src="https://gov-au-ui-kit.apps.staging.digital.gov.au/latest/img/coat-of-arms.png">
      </div>
      <div class="footer-links">
        <nav>
          <ul>
            <li><a href="#">Global link</a></li>
            <li><a href="#">Global link</a></li>
          </ul>
        </nav>
        <p>
          &copy; Commonwealth of Australia <br>
          <a href="https://github.com/AusDTO/gov-au-ui-kit/blob/master/LICENSE.md" rel="external license">
            Licensed under the MIT License.
          </a>
        </p>
      </div>
    </section>`
So that the div will `footer-links` class to be displayed in-line, so as the styles for the crest. 
What is our decision on it. We are having the requirement for in-line list in footer, currently without custom css, we can not have it.